### DISCUSSION TITLE
i jump into other forms when i click next button

### DISCUSSION THREAD

**Discussion by Jericho2525** [filamentphp/filament#14794] [Help]
public static function form(Form $form): Form
    {
        if(request()->route()->getName() == 'filament.admin.resources.applications.create_business'){
            return $form
                ->schema(self::businessForm())->columns(1);
        }else{
            return $form
            ->schema(self::individualForm())->columns(1);
        }
    }

    private static function businessForm(){
        return [  Wizard::make()
        ->schema([
            /*Wizard\Step::make('Subscription Package')
                ->schema(self::SubscriptionPackage())
                ->columns(3),*/
            Wizard\Step::make('CompanyInformation')
            ->schema(self::CompanyInformation()),

            Wizard\Step::make('BankingReferences')
                ->schema(self::BankReferences()),
            Wizard\Step::make('Financial Information')
                ->schema(self::FinancialInformation()),
        ])->submitAction(new HtmlString(Blade::render(<<<BLADE
        <x-filament::button
            type="submit"
            size="sm"
            wire:submit="Save"
        >
            Save
        </x-filament::button>
        BLADE)))->persistStepInQueryString('business'),];
    }
    
      private static function individualForm(){
        return [
            Wizard::make()
                ->schema([
                    /*Wizard\Step::make('Subscription Package')
                        ->schema(self::SubscriptionPackage())
                        ->columns(3),*/
                    Wizard\Step::make('Personal Information')
                    ->schema(ApplicationResource::PersonalInformation()),

                    Wizard\Step::make('Income Information')
                        ->schema(self::IncomeInformation()),
                    Wizard\Step::make('Financial Information')
                        ->schema(self::FinancialInformation()),
                ])->submitAction(new HtmlString(Blade::render(<<<BLADE
                <x-filament::button
                    type="submit"
                    size="sm"
                    wire:submit="Save"
                >
                    Save
                </x-filament::button>
                BLADE))),
            ];
    }

**Comment by leandrocfe** [comment]
What is the issue? Could you explain please?

**Reply by Jericho2525** [reply]
When I click the Next button on the Business form, it jumps to the Individual form instead of staying on the Business form. The form is using a wizard.
